crates/issue/src/lib.rs

```diff
@@ -267,39 +269,37 @@ impl<T: Config> Module<T> {
         );
         ext::collateral::lock_collateral::<T>(&requester, griefing_collateral)?;
 
-        let fee_polkabtc = ext::fee::get_issue_fee::<T>(amount_polkabtc)?;
-        let amount_btc = amount_polkabtc + fee_polkabtc;
-
-        let issue_id = ext::security::get_secure_id::<T>(&requester);
-
         ext::vault_registry::try_increase_to_be_issued_tokens::<T>(&vault_id, amount_btc)?;
 
+        let fee_polkabtc = ext::fee::get_issue_fee::<T>(amount_btc)?;
+        let user_polkabtc = amount_btc
```

Comment:
maybe rename to `amount_issued`

Reply:
That's ambiguous, though - technically the vault issues both the fee and the amount sent to the user

Reply:
`amount_user`?